tests/_support/Page/Framework.php

```diff
@@ -1129,4 +1129,38 @@ public function iUploadTheItemTypeCASEFile()
         return $this;
     }
 
+    /**
+     * @When /^I download the framework PDF file$/
+     */
+    public function iDownloadTheFrameworkPdfFile(): Framework
+    {
+        $I = $this->I;
+
+        $I->click('Export', '#itemInfo');
+        $I->waitForElementVisible('#exportModal a.btn-export-pdf');
+        $I->click('#pdfgen');
+        $url = $I->grabAttributeFrom('#exportModal a.btn-export-case', 'href');
```

Comment:
This line is downloading a CASE json file, not the PDF file, so the expectation from the method name is not met (and we never verify that the PDF can be downloaded).